hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -129,6 +132,10 @@
    * |----------------------------------------------------------------------|
    * |  directoryTable    | parentId/directoryName -> DirectoryInfo         |
    * |----------------------------------------------------------------------|
+   * |  fileTable         | parentId/fileName/id -> FileInfo                |
+   * |----------------------------------------------------------------------|
+   * |  openFileTable     | parentId/fileName -> FileInfo                   |
```

Comment:
openFileTable should have additional id after parentId/fileName and fileTable key should be parentId/fileName.

Reply:
Thanks again. Done in latest commit